app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file_1, file_2 = command.split(" ")
```

Comment:
check that the splitted command consists of three parts before unpacking and that the command is "cp"